data_scientist_skills/data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+import string
+import unidecode
+
+def get_data(df1_dir, df2_dir): #Relative path to each data file
```

Comment:
Can we assume that we can find the data in the raw_data directory? i.e. no need to pass parameters